backend/tests/services/test_docker_services.py

```
@@ -7,20 +7,19 @@
 import pytest
 from fastapi import HTTPException
 
-from backend.docker_utils.containers import (
-    ensure_containers_running,
+from backend.docker_utils.compose_utils import (
+    check_service_health,
+    ensure_services_running,
     get_container_logs,
-    stop_containers,
+    stop_services,
```

Ruff (0.8.2)

`backend.docker_utils.compose_utils.stop_services` imported but unused

Remove unused import: `backend.docker_utils.compose_utils.stop_services`

(F401)